rewrite-java/src/main/java/org/openrewrite/java/tree/TreeBuilder.java

```diff
@@ -283,57 +282,60 @@ public J buildDeclaration(J.ClassDecl insertionScope, String snippet, JavaType..
     }
 
     /**
-     * This generates a String representation of a method declaration with an empty body based on the supplied JavaType.Method.
-     * These stubs can be used to ensure that type attribution succeeds for methods included in snippets built into AST elements.
+     * This method generates a stubbed out method snippet based on the Java Method Type.
```

Comment:
I like the removed version of this comment better because:

* It describes what "stub" means in practical terms. 
* It describes the intent of the function, RE: ensuring type attribution will succeed for methods.
* Both of those attributes of this method aren't obvious from the method signature itself.

On the other hand, it is a lot of words. Do callers _need_ to know all of that?
Reasonable engineers can disagree, but I suspect that on the whole they benefit.